examples/06-suspense-like/src/app.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import { WithStyles, withStyles } from "@material-ui/core/styles";
+import { trackPromise, usePromiseTracker } from "react-promise-tracker";
+import { Quote, getQuote, getPicture } from "./api";
+import { styles } from "./styles";
+
+
+const Suspense: React.FC<{ tag: string, className?: string }> = ({ tag, className, children }) => {
+  const { promiseInProgress } = usePromiseTracker({ area: tag, delay: 0 });
```

Comment:
Maybe it's not necessary feed the `delay` equals 0.

Reply:
Yes, you are right! Didn't realize it was optional.

Reply:
Hummm .... it seems the typing file is not correct then, cause it is not optional there.

Reply:
What a weird thing .... the .d.ts shipped with 2.0 version seems to have this:

```
interface Config {
   area: string;
   delay: number;
}
```

However, in src they are optional:

```
interface Config {
   area?: string;
   delay?: number;
}
```

Is this a bug?

Reply:
I have removed delay param anyway, but we should check the d.ts in v2.0 package as it seems to be not in sync with the one in src. @brauliodiez are you aware of this?

Reply:
I think in current build the are optional, I think it has been published (v 2.0.2)

```
interface Config {
   area?: string;
   delay?: number;
}
```